src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -262,16 +262,16 @@ private function fixPhpDoc(Tokens $tokens, int $index, array $uses, string $name
     {
         $phpDoc = $tokens[$index];
         $phpDocContent = $phpDoc->getContent();
-        Preg::matchAll('#@([^\s]+)\s+([^\s]+)#', $phpDocContent, $matches);
+        Preg::matchAll('#@([^\s]+)(\s+)((?:[a-zA-Z0-9_\\\\]+)+)#', $phpDocContent, $matches);
 
-        if ([] !== $matches) {
-            foreach ($matches[2] as $i => $typeName) {
+        if ([] !== $matches[0]) {
+            foreach ($matches[3] as $i => $typeName) {
                 if (!\in_array($matches[1][$i], ['param', 'return', 'see', 'throws', 'var'], true)) {
```

Comment:
phpdoc like `phpstan-param` should be supported

Reply:
It's a bug fix, no new features in this PR.

Reply:
Covered in #7628

Reply:
Great!